Cargo.toml

```diff
@@ -1,9 +1,11 @@
 [workspace]
 members = [
 	"docbrown/core",
-	"docbrown/db"
+	"docbrown/db",
+	"examples/btc",
```

Comment:
as we talked, have a look at the examples structure and the version with multiple binaries